core/src/main/java/org/apache/iceberg/BaseOverwriteFiles.java

```diff
@@ -35,7 +35,7 @@
   private Long startingSnapshotId = null;
   private Expression conflictDetectionFilter = null;
   private boolean validateNewDataFiles = false;
-  private boolean validateNewDeleteFiles = false;
+  private boolean validateNewDeletes = false;
```

Comment:
Why the rename here?

Reply:
Cause we not only validate added delete files but also deleted data files. We have two types of deletes to cover.